File: facade/HotelReservation.java
package facade;

public class HotelReservation {

    public void reserveRoom(String roomId, String checkInDate, String checkOutDate) {
        System.out.println("Reserving the room: " + roomId + " with checkInDate: " + checkInDate
                + " and checkOutDate: " + checkOutDate);
    }
}

File: PHP/src/Structural/Facade/HotelReservation.php
<?php

namespace App\Structural\Facade;

class HotelReservation
{    
    /** 
     * @param  string $roomId
     * @param  string $checkInDate
     * @param  string $checkOutDate
     * @return void
     */
    public function reserveRoom(string $roomId, string $checkInDate, string $checkOutDate): void
    {
        echo ("Reserving the room: " . $roomId . " with checkInDate: " . $checkInDate
            . " and checkOutDate: " . $checkOutDate);
    }
}
